framework/src/outputs/Output.C

```diff
@@ -114,6 +116,10 @@ Output::Output(const InputParameters & parameters)
                                            : -std::numeric_limits<Real>::max()),
     _end_time(isParamValid("end_time") ? getParam<Real>("end_time")
                                        : std::numeric_limits<Real>::max()),
+    _start_step(isParamValid("start_step") ? getParam<int>("start_step")
+                                           : -std::numeric_limits<int>::max()),
```

Comment:
This code can now be updated `std::numeric_limit<int>::lowest`

Reply:
OK, I'll change this and the one above it.